downloads.py

```diff
@@ -97,7 +98,7 @@ def get_pmgr(plugins_list):
     for repo_file in os.listdir(repo_dir):
         with open(os.path.join(repo_dir, repo_file)) as fhd:
             content = json.loads(fhd.read())
-            jsonschema.validate(content, schema)
+#            jsonschema.validate(content, schema)
```

Comment:
Why disabling schema validation?

Reply:
By accident. (Didn't get my pip to work...)